app.py

```diff
@@ -109,26 +109,33 @@ def signing_up():
     return redirect(url_for('sign_in'))
 
 
+@app.get('/logout')
+def logout():
 
+    session.pop('user_id', None)
+    return redirect(url_for('sign_in'))
```

Comment:
You could add a flash message saying "Successful Logout" or something along those lines.